tests/execution.rs

```diff
@@ -15,18 +15,18 @@
 // specific language governing permissions and limitations
```

Comment:
I renamed the tests from 'extensions' to 'execution' to better reflect it is testing execution in general rather than only extension execution